spring-batch-docs/asciidoc/spring-batch-intro.adoc

```diff
@@ -6,8 +6,6 @@
 
 == Spring Batch Introduction
 
-include::toggle.adoc[]
```

Comment:
There is no code sample in this section, so I think there is no need for the toggle here.